doc/releases/changelog-dev.md

```diff
@@ -183,134 +182,149 @@
-  ZZ = qml.PauliZ(0) @ qml.PauliZ(1)
+* A new function called `load_basisset` has been added to extract `qchem` basis set data from the Basis Set Exchange
+  library.
+  [(#3363)](https://github.com/PennyLaneAI/pennylane/pull/3363)
 
-  H =  2 * XX + f1 * YY + f2 * ZZ
-  ```
-  ```pycon
-  >>> H
-  ParametrizedHamiltonian: terms=3
-  >>> params = [1.2, [2.3, 3.4]]
-  >>> H(params, t=0.5)
-  (2*(PauliX(wires=[1]) @ PauliX(wires=[1]))) + ((-0.2876553535461426*(PauliY(wires=[0]) @ PauliY(wires=[0]))) + (1.5179612636566162*(PauliZ(wires=[0]) @ PauliZ(wires=[1]))))
+* A new function called `max_entropy` has been added to compute the maximum entropy of a quantum state.
+  [(#3594)](https://github.com/PennyLaneAI/pennylane/pull/3594)
+
+* A new template called `TwoLocalSwapNetwork` has been added that implements a canonical 2-complete linear (2-CCL) swap network
+  described in [arXiv:1905.05118](https://arxiv.org/abs/1905.05118).
+  [(#3447)](https://github.com/PennyLaneAI/pennylane/pull/3447)
+
+  ```python3
+  dev = qml.device('default.qubit', wires=5)
+  weights = np.random.random(size=TwoLocalSwapNetwork.shape(len(dev.wires)))
+  acquaintances = lambda index, wires, param: (qml.CRY(param, wires=index)
+                                   if np.abs(wires[0]-wires[1]) else qml.CRZ(param, wires=index))
+  @qml.qnode(dev)
+  def swap_network_circuit():
+     qml.templates.TwoLocalSwapNetwork(dev.wires, acquaintances, weights, fermionic=False)
+     return qml.state()
   ```
-  The same `ParametrizedHamiltonian` can also be constructed via a list of coefficients and operators:
 
   ```pycon
-  >>> coeffs = [2, f1, f2]
-  >>> ops = [XX, YY, ZZ]
-  >>> H =  qml.ops.dot(coeffs, ops)
+  >>> print(weights)
+  tensor([0.20308242, 0.91906199, 0.67988804, 0.81290256, 0.08708985,
+          0.81860084, 0.34448344, 0.05655892, 0.61781612, 0.51829044], requires_grad=True)
+  >>> qml.draw(swap_network_circuit, expansion_strategy = 'device')()
+  0: ─╭●────────╭SWAP─────────────────╭●────────╭SWAP─────────────────╭●────────╭SWAP─┤  State
+  1: ─╰RY(0.20)─╰SWAP─╭●────────╭SWAP─╰RY(0.09)─╰SWAP─╭●────────╭SWAP─╰RY(0.62)─╰SWAP─┤  State
+  2: ─╭●────────╭SWAP─╰RY(0.68)─╰SWAP─╭●────────╭SWAP─╰RY(0.34)─╰SWAP─╭●────────╭SWAP─┤  State
+  3: ─╰RY(0.92)─╰SWAP─╭●────────╭SWAP─╰RY(0.82)─╰SWAP─╭●────────╭SWAP─╰RY(0.52)─╰SWAP─┤  State
+  4: ─────────────────╰RY(0.81)─╰SWAP─────────────────╰RY(0.06)─╰SWAP─────────────────┤  State
   ```
-  
+
 <h3>Improvements</h3>
 
-* Most channels in are now fully differentiable in all interfaces.
+* `qml.math.matmul` now supports PyTorch and Autograd tensors.
+  [(#3613)](https://github.com/PennyLaneAI/pennylane/pull/3613)
+
+* `qml.math.size` now supports PyTorch tensors.
+  [(#3606)](https://github.com/PennyLaneAI/pennylane/pull/3606)
+
+* `qml.purity` can now be used as a measurement process.
+  [(#3551)](https://github.com/PennyLaneAI/pennylane/pull/3551)
+
+* Most quantum channels are now fully differentiable on all interfaces.
```

Comment:
Worth being more specific here? Can we give a list of unsupported quantum channels?